.github/workflows/gh-ci.yaml

```diff
@@ -23,31 +23,18 @@ defaults:
     shell: bash  -l {0} 
 
 jobs:
-  environment-config:
-    runs-on: ubuntu-latest
-    outputs:
-      stable-python-version: ${{ steps.get-compatible-python.outputs.stable-python }}
-      python-matrix: ${{ steps.get-compatible-python.outputs.python-versions }}
-    steps:
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.11"
-
-      - id: get-compatible-python
-        uses: MDAnalysis/mdanalysis-compatible-python@main
-        with:
-          release: "latest"
-
   main-tests:
     if: "github.repository == 'MDAnalysis/waterdynamics'"
-    needs: environment-config
     runs-on: ${{ matrix.os }}
     strategy:
         fail-fast: false
         matrix:
           os: [macOS-latest, ubuntu-latest, windows-latest]
-          python-version: ${{ fromJSON(needs.environment-config.outputs.python-matrix) }}
+          python-version: ["3.10", "3.11", "3.12"]
           mdanalysis-version: ["latest", "develop"]
+          exclude:
+            - python-version: "3.12"
+              mdanalysis-version: "latest"
```

Comment:
Possibly add comment here to tell future maintainers how to handle "latest" and highest Python versions.

Reply:
Done